modules/ml-training/k8s-managed/images/pytorch-mnist/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# We need to use the nvcr.io/nvidia/pytorch image as a base image to support both linux/amd64 and linux_arm64 platforms.
+# PyTorch=1.13.0, cuda=11.8.0
+# Ref: https://github.com/kubeflow/katib/tree/master/examples/v1beta1/trial-images/pytorch-mnist
+FROM 763104351884.dkr.ecr.eu-central-1.amazonaws.com/pytorch-training:2.1.0-gpu-py310-cu121-ubuntu20.04-ec2
```

Comment:
When I run this in us-east-1, I cannot pull this image......I think this cannot be hardcoded (or be region specific??)